### ISSUE TITLE
Is VT100 working out for you?

### ISSUE THREAD

**Issue by Keno** [JuliaImages/ImageInTerminal.jl#3]
I saw you added VT100 as a test dependency. Please let me know if you run into any trouble. So far I've only really used it for testing my own packages, so it's probably less well documented for other people than it should be. Happy to make any improvements to make it work for you.

**Comment by Evizero** [comment]
Hi. Thanks! Very nice of you drop by. I haven't pushed the latest changes I made (writing test)  as I am traveling for work. So far I got away by testing functions the "normal" way.

What I want to use VT100 for is simulating terminals of various sizes, since the output of this package depends on that and `isinteractive()`. I though that would a cool way to start properly testing terminal-based packages. (if you look at the tests of UnicodePlots.jl you see why it is time I think about it more seriously) .

I have already looked through the code of VT100 a little and hope I'll get there step by step (I am optimistic). I will let you know if I hit a brick wall.
